### DISCUSSION TITLE
New to GitHub and no programming experience

### DISCUSSION THREAD

**Discussion by apadro** [community/community#23943] [New to GitHub]
<p>Hello!! Im new to GitHub and no programming experience. I need to download and install in my Mac mini, Big Sur an SDRPlay Dump1090 program. but I don’t know how to download it and install it. Please guide me on how to do this!!!</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>If you’re looking for help with a project that is hosted on GitHub, typically you’ll find instructions on the main repository page - if it just looks like a bunch of files, keep scrolling and you’ll find a README file.</p>
<p>If you have questions about their project, you’ll want to open an issue in their repository, or follow whatever instructions they have for contacting them.</p>

**Comment by apadro** [comment, marked as answer]
<p>What I’m confused it’s what steps I need to take to be able to install that program. For what I understand it’s that program needs to be compiled. I followed the steps but I get errors that the operant or program or that argument used it’s not presentI or not the correct one. Mind you, I’m not a programmer. The instructions are a bit confusing.</p>

**Comment by DevRehannaLP** [comment]
Since I started with GitHub I've had nothing but confusion and let downs.  I've had someone from Microsoft hack my computer and lead me down a road of even more confusion.  At times I log into VS code and my source is telling me I have 10K sources which how that is possible in just a month of using GitHub I have no idea.  But all I am trying to do is learn and learn the correct way.  I don't feel I've been lead correctly as of yet.  I keep reaching out and I'm lead to only get help from an AI called Copilot which seems to be leading me in an even more wrong way.  So Here I am reaching out here wondering if I can get help from anyone willing to explain all this chaos to me.  If not, I'm not upset in anyway and just hope Git or GitHub will not be when I'm driven to the point on deleting every repository I have, deleting every file in my Dev created Drive for coding and delete anything that's been done this far because I'm stuck believing the only way out of this choas is to delete everything and either start over and just never deal with coding again since day one it's been nothing but headaches and stress. -shrugs-
